src/app/_variables.scss

```diff
@@ -147,3 +141,51 @@ produces
 @function dim-rgb-values-to-rgba($values, $alpha: 1) {
   @return #{'rgba(' + $values + ', ' + $alpha + ')'};
 }
+
+// Theme variables
+:root {
```

Comment:
Gotcha, I missed that description at the top of the file.

Will move in `main.scss`

Reply:
Opted to partition the customisable variables into a new `_theme.scss` in order to better separate what can be themes from the system-level component styles (tooltip corners/arrows) that shouldn't be customisable. These have been kept in `_variables.scss` as sass variables to prevent unwanted changes.